bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/AbstractOptionsFieldImpl.java

```diff
@@ -51,6 +52,18 @@ public boolean isEnforceEnum() {
         return enforceEnum;
     }
 
+    public String[] removeDuplicates(boolean flag, int length) {
```

Comment:
What do you mean by flag ? what is length ? Create a function which manipulates the map via streams.